tests/foursixnine/desktop_stuff.pm

```diff
@@ -0,0 +1,41 @@
+
+# Copyright (C) 2015-2018 SUSE Linux Products GmbH
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Shut down the system
+# - Poweroff system
+# Maintainer: Oleksandr Orlov <oorlov@suse.de>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use utils;
+use x11utils 'ensure_unlocked_desktop';
+
+sub run {
+    select_console('user-console');
+    script_run('for key in $(gsettings list-keys org.gnome.desktop.session); do echo $key $(gsettings get org.gnome.desktop.session $key); done');
```

Comment:
As you are already reading out the settings you could set a very short idle time and only sleep for that time. If you want to be strict you can assert that the previously set value is a sane default

Reply:
Nah, this test module, is actually for testing purposes

Reply:
Yeah I got that but we want to have an "explicit test for screensaver" nevertheless in the future, right? And I think a valid test would be:

* assert that screensaver is enabled by default with sane timeout
* set screensaver to low value N seconds
* assert screensaver activates after N seconds
* disable screensaver for good to let other test modules not care about screensaver at all